terracumber/terraformer.py

```diff
@@ -56,7 +60,13 @@ def inject_repos(self, custom_repositories_json):
                     replacement = ''.join(replacement_list)
                     placeholder = '//' + node + '_additional_repos'
                     for line in fileinput.input("%s/main.tf" % self.terraform_path, inplace=True):
-                        print(line.rstrip().replace(placeholder, replacement))
+                        (new_line, n) = subn(placeholder, replacement, line)
+                        print(new_line, end='')
+                        n_replaced += n
+            if n_replaced != 1:
+                print("ERROR: please use exactly 1 placeholder for additional repositories in main.tf")
```

Comment:
As you mentioned, all the error management is at `terracumber-cli`, so not adding it there?

The idea of having everything in terracumber-cli is so we can use the code at the `terracumber` code from a different interface, and always print the errors, there, or send them to a file, or to a website, or whatever.

Reply:
ok i also prefer it at the `terracumber-cli` file.

i don't like one high level file knowing about the details of one low level file but ok. I'll break the separation of resposibilities here.

Reply:
done